stack2stack.py

```diff
@@ -93,27 +94,55 @@ def migrate_images():
     new_cloud_glance_client = glance_client.Client('1', token=new_cloud_keystone_client.auth_token, endpoint=new_endpoint.publicurl)
     images = old_cloud_glance_client.images.list()
     for i in images:
-        print 'Found image %s' % i.name
+        print ''
+        print 'Old Cloud: Found image           - %s' % i.name
+        print '           Image ID              - %s' % i.id
+        try:
+            print '           Image       Type      - %s' % i.properties['image_type']
+        except KeyError:
+            print '           Image       Type      - image'
```

Comment:
As we only deal with images (not snapshots) this can be refactored to

```python
image_type = 'image'
if i.has_key('image_type'):
    image_type = i.properties['image_type']
print '           Image       Type      - %s' % image_type